backend/test/mastodon/statuses.spec.ts

```diff
@@ -60,9 +61,9 @@ describe('Mastodon APIs', () => {
 			assert.equal(res.status, 200)
 			assertJSON(res)
 
-			const data = await res.json<any>()
-			assert(data.uri.includes('example.com'))
-			assert(data.uri.includes(data.id))
+			const data = await res.json<MastodonStatus>()
+			assert((data.uri as unknown as string).includes('example.com'))
```

Comment:
I hate this because the type says URL but clearly the actual runtime value is a string.
How about instead of a cast we fudge it with:

```
			assert((data.uri.toString()).includes('example.com'))
```

Or perhaps it is best to leave the ` as unknown as string` to remind us that the compile time and runtime types don't line up?

Reply:
yeah this type of thing is quite confusing/messed-up 😭

I am not sure, whatever you think is best works for me 🙂👍 

but if you want a reminder then I think I would the go for something like:
```suggestion
                         //@ts-expect-error -- TO FIX: uri should be of type URL but in the actual runtime is a string
			assert((data.uri as unknown as string).includes('example.com'))
```